variation/tokenizers/deletion_range_base.py

```diff
@@ -0,0 +1,59 @@
+"""A module for tokenizing genomic deletion ranges."""
+from typing import Dict
+from .tokenizer import Tokenizer
+from variation.schemas.token_response_schema import TokenMatchType
+from abc import abstractmethod
+
+
+class DeletionRangeBase(Tokenizer):
+    """The tokenizer class for genomic deletion range."""
+
+    def __init__(self) -> None:
+        """Initialize the Genomic Deletion Range Class."""
+        self.parts = None
+
+    def match(self, input_string: str):
```

Comment:
Does this return like an Optional[Token] or something to that effect?

Reply:
Yes, `Optional[DeletionRange]`